src/app/components/home/home.component.html

```diff
@@ -1,146 +1,188 @@
-<section id="header">
-  <div class="container">
-    <div class="row justify-content-md-center mt-4 text-center">
-      <div class="cb-image">
-        <img src="../../../assets/home/logo_bw.png" width="100" alt="Logo">
+<!DOCTYPE html>
```

Comment:
You cannot add html, head and body tags in a component. 
Please make sure you follow the syntax and principles
Thanks